cli/stubs/z-performance.ini

```diff
@@ -67,3 +67,4 @@ xdebug.remote_host=localhost
 xdebug.remote_port=9000
 xdebug.remote_autostart=1
 xdebug.idekey=PHPSTORM
+xdebug.max_nesting_level=2000
```

Comment:
at the comment you wrote 200, but here you set it to 2000

Reply:
I don't mind just setting this to `-1`.

Reply:
@mischabraam that has been updated.